CodeEdit/Commands/CommandPaletteView.swift

```diff
@@ -0,0 +1,285 @@
+//
+//  CommandPaletteView.swift
+//  CodeEdit
+//
+//  Created by Alex Sinelnikov on 24.05.2022.
+//
+
+import SwiftUI
+import Keybindings
+import CodeEditUI
+
+public struct CommandPaletteView: View {
```

Comment:
Please add proper documentation to `public` classes, structs, methods and variables/constants.

A guide about code documentation can be found [here](https://nshipster.com/swift-documentation/).